gno.land/pkg/gnoland/app.go

```diff
@@ -161,6 +161,10 @@ func InitChainer(baseApp *sdk.BaseApp, acctKpr auth.AccountKeeperI, bankKpr bank
 			// Get genesis state
 			genState := req.AppState.(GnoGenesisState)
 
+			// Init ZeroAccount
+			zeroAcc := acctKpr.NewAccountWithAddress(ctx, std.ZeroAddress())
+			acctKpr.SetAccount(ctx, zeroAcc)
```

Comment:
if we decide keeping ZeroAddress official, then I think it makes sense to register the account by default with `account_id: 0` 👍